package.json

```diff
@@ -102,8 +106,10 @@
     "faucet": "0.0.1",
     "istanbul": "1.1.0-alpha.1",
     "jsonpath": "1.0.1",
+    "license-checker": "25.0.1",
```

Comment:
Shouldn't these be dev dependencies?

Reply:
I think you'll find they are - perhaps the diff squashed the start of the `devDependencies` away.